website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -2737,6 +2737,14 @@ This endpoints adds one or more tags to the workspace. The workspace must alread
 tag element that supplies an `id` attribute must exist. If the `name` attribute is used, and no
 matching organization tag exists with such name, a new one will be created.
 
+Here are some things to note about tags:
```

Comment:
Can you please apply my above suggestion to this repeat section? Also maybe we should consider just link to the tags page instead of repeating this information?